# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_sh-utils.patch-x
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./lib/getline.c.orig	Wed May  1 12:05:24 2002
+++ ./lib/getline.c	Wed May  1 12:05:24 2002
@@ -22,10 +22,8 @@
 # include <config.h>
 #endif
 
-/* The `getdelim' function is only declared if the following symbol
-   is defined.  */
-#ifndef _GNU_SOURCE
-# define _GNU_SOURCE 1
+#ifdef _GNU_SOURCE
+# undef _GNU_SOURCE
 #endif
 
 #include <stdio.h>
--- ./lib/getline.h.orig	Thu May 13 22:51:26 1999
+++ ./lib/getline.h	Wed May  1 12:05:24 2002
@@ -17,6 +17,10 @@
 #ifndef GETLINE_H_
 # define GETLINE_H_ 1
 
+#ifdef _GNU_SOURCE
+# undef _GNU_SOURCE
+#endif
+
 # include <stdio.h>
 
 # ifndef PARAMS
--- ./config.h.in.orig	Wed May  1 12:10:17 2002
+++ ./config.h.in	Wed May  1 12:10:36 2002
@@ -123,12 +123,6 @@
    byte first (like Motorola and SPARC, unlike Intel and VAX).  */
 #undef WORDS_BIGENDIAN
 
-/* This is always defined.  It enables GNU extensions on systems that
-   have them.  */
-#ifndef _GNU_SOURCE
-# undef _GNU_SOURCE
-#endif
-
 /* Define if your system's definition of `struct termios' has a member
    named c_line.  */
 #undef HAVE_C_LINE
